implementations/python/sugar-lift-py-tests/tests/test_census_construction_door.py

```
@@ -0,0 +1,61 @@
+        encoding="utf-8",
+    )
+    # Honest door: typed residual, never unconditional RuntimeSelected.
+    reporter = CollectingReporter()
+    sf = open_source_file_for_construction(
+        path, root=tmp_path, reporter=reporter, populate_derived=True
+    )
+    for fn in sf.functions():
+        try:
+            fn.sugar()
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)